tapir/static/css/base.css

```diff
@@ -57,4 +57,44 @@ body {
 .button-icon {
   font-size: 16px;
   vertical-align: -2px;
-}
\ No newline at end of file
+}
+
+/*
+The fonts are from https://github.com/jossef/material-design-icons-iconfont/releases/tag/v6.1.0
+*/
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url(/static/material-icons/MaterialIcons-Regular.eot); /* For IE6-8 */
+  src: local('Material Icons'),
+    local('MaterialIcons-Regular'),
+    url(/static/material-icons/MaterialIcons-Regular.woff2) format('woff2'),
+    url(/static/material-icons/MaterialIcons-Regular.woff) format('woff'),
+    url(/static/material-icons/MaterialIcons-Regular.ttf) format('truetype');
+}
+
+.material-icons {
```

Comment:
Did you copy-paste this from somewhere? If so, please put the source here in case somebody ever has to touch it :)